### PR TITLE
Contributor documentation

### PR DESCRIPTION
**Pull request by drphilmarshall**
@swagnercarena Look! My first paltas pull request. I added a short section to the README about how to contribute to paltas, see what you think. (I also noticed that tensorflow was not on the requirements list, so added that.)

### CODE REVIEW

File: README.rst
Comment (swagnercarena) [inline README.rst:91]: I think this is mostly correct, besides the fact that our standard is to use Args instead of Parameters in our docstrings. Until we write our own contribution description (which we can hold off on if / until we have a larger community using paltas) this looks good.

File: requirements.txt
Comment (swagnercarena) [inline requirements.txt:12]: The tensorflow requirement can be hefty for some users, especially if they aren't intending to use the analysis functions of paltas. There is currently a note in the readme about installing tensorflow if you want to use the analysis functions. It would be best to keep this out. We may want to be more explicit about separating the analysis features moving forward.

**Review by swagnercarena — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (drphilmarshall) [reply requirements.txt]: OK! Removed this line.

File: README.rst
Comment (drphilmarshall) [reply README.rst:91]: Regarding args vs parameters, do we follow [Google docstring style](https://www.datacamp.com/tutorial/docstrings-python)? I can add a short statement with a pointer to the right style guide.

**Comment by drphilmarshall** [comment]
@swagnercarena I think I am all done!

**Review by swagnercarena — APPROVED** [review]
Looks great!

**Comment by swagnercarena** [comment]
Once you merge / pull the changes in the main branch into this branch, you should see that all the tests are passing and we can merge this into main.

**Comment by drphilmarshall** [comment]
Done! Nice that GitHub already knew (somehow) that the tests would pass, so didn't even need to run them... See below for screenshot. No tests required, apparently!

image

**Pull request merged**
